examples/ssx-test-nextauth-email/README.md

```diff
@@ -0,0 +1,44 @@
+# SSX NextAuth with Email Example
+
+This is an example dapp for ssx-react. It's a Next.js app that implements a login flow with SSX and Email. The user is allowed to sign in with email at any time, and with SSX if the email is linked to a web3 address. It was generated from create from [RainbowKit's Create Dapp template](https://www.rainbowkit.com/) and then modified to use ssx-react for signing in.
```

Comment:
To change around the following (but feel free to also mention details about how it was set up / add details about how the example is using prisma): 

This is an example dapp for `ssx-react`. It's a Next.js app that implements a login flow with SSX and RainbowKit, but requires a user to first log in using their email. 
- First a user signs in with email 
- Next, a user associates their Ethereum account to their login information
- Finally, they can use that associated Ethereum account to Sign-In with Ethereum